samples/excel/89-preview-apis/events-worksheet-activated.yaml

```diff
@@ -0,0 +1,156 @@
+order: 7
+id: excel-events-worksheet-activated
+name: Events - Worksheet activated
+description: Add an event handler on the worksheet activated and deactivated events
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#activate-event-handler").click(() => tryCatch(onActivateEventHandler));
+        $("#deactivate-event-handler").click(() => tryCatch(onDeactivateEventHandler));
+        $("#delete-worksheet").click(() => tryCatch(deleteWorksheet));
+
+        async function onActivateEventHandler() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets;
+                sheet.onActivated.add(onActivate);
```

Comment:
The line
 sheet.onActivated.add(onActivate); 
 is an example where it gets misleading to use "sheet" singular for the collection of worksheets.. It is the _collection_ that has an onActivated property, not some one worksheet.